Automattic/jetpack
# Revert #33394 #33036 and #33042 - VP conditional asset enqueueing

Revert #33394 #33036 #33042 

Fixes #33660

## Proposed changes:
This PR reverts the changes on the mentioned PRs. As noted on the #33660 the fix won't handle feeds/query loops properly.

This will trigger a regression of #32235 

### Other information:


## Jetpack product discussion
p1697665115617289-slack-C03TA48NSUX


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* On a site with Jetpack connected and a clsasic theme like 2016
* Create a post with a VideoPress video on it.
* Create a second post with no video on it.
* Check the homepage (the post list) and confirm the video of the 2nd post is shown. (That is the first post you created with this instructions, not the last one. )

(use dasprofilen.wordpress.com if you need a site ready for test case)

### Before

<img width="996" alt="image" src="https://github.com/Automattic/jetpack/assets/746152/b2ef70fa-07d9-4023-b82b-ff524598fd98">

## After
<img width="633" alt="image" src="https://github.com/Automattic/jetpack/assets/157240/1604ecde-d980-404b-ac89-c50bcc130343">
